src/utils/goma.js

```diff
@@ -145,14 +145,22 @@ function authenticateGoma(config) {
 
   if (!gomaIsAuthenticated()) {
     console.log(color.childExec('goma_auth.py', ['login'], { cwd: gomaDir }));
-    childProcess.execFileSync('python', ['goma_auth.py', 'login'], {
+    const { status, error } = depot.spawnSync('python', ['goma_auth.py', 'login'], {
       cwd: gomaDir,
       stdio: 'inherit',
       env: {
         ...process.env,
         AGREE_NOTGOMA_TOS: '1',
       },
     });
+
+    if (status !== 0) {
+      console.error(
+        `${color.err} Failed to run command, exit code was "${status}", error was '${error}'`,
```

Comment:
Could we do something more like: https://github.com/electron/build-tools/blob/a7e1b97325ea37947db452605406c85b3ca5b9dd/src/e#L219-L224 here?